### ISSUE TITLE
Python 2.6 compatibility

### ISSUE THREAD

**Issue by larsweiler** [willthames/ansible-lint#304]
# Issue Type
- Bug report

# Ansible and Ansible Lint details

```
ansible --version
ansible 2.4.1.0
ansible-lint --version
Python Error (but it's 3.4.17)
```

- ansible installation method: one of source, pip, OS package
OS package (RHEL 6.9 EPEL)
- ansible-lint installation method: one of source, pip, OS package
newly built package from https://koji.fedoraproject.org/koji/buildinfo?buildID=1005669

# Desired Behaviour

It should not quit with a Python error.

# Actual Behaviour (Bug report only)

Currently it just quits with an error.

```
ansible-lint
Traceback (most recent call last):
  File "/usr/bin/ansible-lint", line 9, in <module>
    load_entry_point('ansible-lint==3.4.17', 'console_scripts', 'ansible-lint')()
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 299, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 2229, in load_entry_point
    return ep.load()
  File "/usr/lib/python2.6/site-packages/pkg_resources.py", line 1948, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/usr/lib/python2.6/site-packages/ansiblelint/__init__.py", line 28, in <module>
    import ansiblelint.utils
  File "/usr/lib/python2.6/site-packages/ansiblelint/utils.py", line 495
    {'include', 'include_tasks', 'import_tasks'}.isdisjoint(task.keys())]
              ^
SyntaxError: invalid syntax
```

The fix is changing line 495 in utils.py to
`set(['include', 'include_tasks', 'import_tasks']).isdisjoint(task.keys())]`

I tested this in https://github.com/larsweiler/ansible-lint/blob/927d84ff4fff1cef12b7c3bd4b10362747d5ef7f/lib/ansiblelint/utils.py#L495

**Comment by willthames** [comment]
If you have a fix, could you turn the fix into a pull request?

**Comment by larsweiler** [comment]
Sure, pull request #305 created.

**Comment by willthames** [comment]
Closed - thanks for raising #305
